starters/vue3-xstate-css/.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I think we should add the prettier rules here

Reply:
I have a separate ticket for doing this - #508 :) Just went ahead and let the CLI install it here since I figured that would be easier. lol